### ISSUE TITLE
forge script error

### ISSUE THREAD

**Issue by wuchuank** [foundry-rs/foundry#2085]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (024e3ed 2022-06-23T03:30:48.7987658Z)

### What command(s) is the bug in?

forge script

### Operating System

Windows

### Describe the bug

Facing an issue when it comes to using forge script. Replicated the solidity scripting tutorial in the foundry book for testing purposes and am still facing the same error

Running `forge script script/NFT.s.sol:MyScript --fork-url http://localhost:8545 --private-key $PRIVATE_KEY_ANVIL` outputs 

```
[⠊] Compiling...
[⠆] Compiling 1 files with 0.8.14
[⠰] Solc 0.8.14 finished in 8.85s
Compiler run successful
The application panicked (crashed).
Message:  Target not found?
Location: cli\src\cmd\forge\script\build.rs:163

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Able to bypass this error by omitting the script name by running `forge script script/NFT.s.sol --fork-url http://localhost:8545 --private-key $PRIVATE_KEY_ANVIL` instead. 

### Steps to Reproduce Error
1. Clone my sample repo [here](https://github.com/wuchuank/foundry-script)
2. Run anvil and add PRIVATE_KEY_ANVIL as an environment variable, alternatively just copy and paste from anvil into the command
3. Run `forge script script/NFT.s.sol:MyScript --fork-url http://localhost:8545 --private-key $PRIVATE_KEY_ANVIL`

**Comment by onbjerg** [comment]
Can you check the nightly that will be released in a few hours?

**Comment by wuchuank** [comment]
Definitely, I'll check it once I wake up in the morning. Thanks for the quick response.

**Comment by wuchuank** [comment]
Just updated foundry to `forge 0.2.0 (28fa0b8 2022-06-25T05:27:08.3962023Z)` and and I'm still getting the same error output 
```
The application panicked (crashed).
Message:  Target not found?
Location: cli\src\cmd\forge\script\build.rs:168

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by onbjerg** [comment]
I'm working on a fix in #2132 but it's gonna take a little while since none of us use Windows. Going to set up a Windows VM today to tighten feedback loop and hopefully we'll have a fix this week :)
